tests/integration/targets/wafv2/tasks/create_webacl.yml

```diff
@@ -1,151 +1,230 @@
+          - name: rate-limit-per-IP
+            priority: 3
+            action:
+              block:
+                custom_response:
+                  response_code: 429
+                  custom_response_body_key: too_many_requests
+            statement:
+              rate_based_statement:
+                limit: 5000
+                aggregate_key_type: IP
+            visibility_config:
+              sampled_requests_enabled: yes
+              cloud_watch_metrics_enabled: yes
+              metric_name: waf-acl-rule-rate-limit-per-IP
+        custom_response_bodies:
+          too_many_requests:
+            content_type: APPLICATION_JSON
+            content: '{ message: "Your request has been blocked due to too many HTTP requests coming from your IP" }'
+        tags:
+          A: B
+          C: D
+      register: ACL
+
+    - name: verify create
+      assert:
+        that:
+          - ACL is changed
+          - ACL.web_acl.name == web_acl_name
+          - not ACL.web_acl.visibility_config.sampled_requests_enabled
+          - ACL.web_acl.rules | count == 3
+          - ACL.web_acl.description == 'hallo eins'
+
+    - name: immutable create web acl
+      wafv2_web_acl:
+        name: "{{ web_acl_name }}"
+        state: present
+        description: hallo eins
+        scope: REGIONAL
+        default_action: Allow
+        sampled_requests: no
+        cloudwatch_metrics: yes
+        metric_name: blub
+        rules:
+          - name: zwei
+            priority: 2
+            action:
+              block: {}
+            visibility_config:
+              sampled_requests_enabled: yes
+              cloud_watch_metrics_enabled: yes
+              metric_name: ddos
+            statement:
+              xss_match_statement:
+                field_to_match:
+                  body: {}
+                text_transformations:
+                  - type: NONE
+                    priority: 0
+          - name: admin_protect
+            priority: 1
+            override_action:
+              none: {}
+            visibility_config:
+              sampled_requests_enabled: yes
+              cloud_watch_metrics_enabled: yes
+              metric_name: fsd
+            statement:
+              managed_rule_group_statement:
+                vendor_name: AWS
+                name: AWSManagedRulesAdminProtectionRuleSet
+          - name: rate-limit-per-IP
+            priority: 3
+            action:
+              block:
+                custom_response:
+                  response_code: 429
+                  custom_response_body_key: too_many_requests
+            statement:
+              rate_based_statement:
+                limit: 5000
+                aggregate_key_type: IP
+              visibility_config:
+                sampled_requests_enabled: yes
+                cloud_watch_metrics_enabled: yes
+                metric_name: waf-acl-rule-rate-limit-per-IP
+        custom_response_bodies:
```

Comment:
With the requested changes above, the integration test pass for me locally.  
What I also like to see is a change, that also removes the `custom_response_bodies`

Reply:
Did add another test now.